### PR TITLE
Add styleSet example for global component

### PR DESCRIPTION
**Pull request by TotallyInformation**

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -3353,7 +3353,7 @@ __Examples__
 Ractive.sharedSet( '_', lodash );
 ```
 
-## Ractive.styleSet()
+## <a name="styleset"></a>Ractive.styleSet()
```
Comment (fskreuz) [inline docs/api.md]: MkDocs automatically generates linked headers based on the header text.

File: docs/api.md [hunk 2]
```diff
@@ -3382,6 +3382,11 @@ __Examples__
 Ractive.styleSet( 'colors.fg', '#000' );
 ```
 
+```js
+// To change a component cssData property
+Ractive.components.MyComponentName.styleSet( 'colors.fg', '#000' );
```
Comment (fskreuz) [inline docs/api.md]: Might be better to show that `MyComponentName` is a custom component that came from `Ractive.extend()` than making assumptions as to what `Ractive.components.MyComponentName` is.

```js
const MyComponent = Ractive.extend({ ... })

// Change styles for all instances of MyComponent
MyComponent.styleSet('colors.fg', '#000')
```

File: docs/api.md (hunk 1 above)
Comment (TotallyInformation) [reply docs/api.md]: Argh, sorry, mistake. I'll redo it.

File: docs/api.md (hunk 2 above)
Comment (TotallyInformation) [reply docs/api.md]: OK, good point, will redo.

Comment (TotallyInformation) [reply docs/api.md]: Actually though, that wouldn't add the component to the global registry would it?

How about:

```js
// Create global component
Ractive.components.MyComponent = Ractive.extend({ ... })
// Change styles for all instances of MyComponent
Ractive.components.MyComponent.styleSet('colors.fg', '#000')
```

As the point was to show a completely different entry point for styleSet.

Comment (fskreuz) [reply docs/api.md]: Not all workflows use the global registry 😉 For instance, I use ES modules and Rollup and the code ressembles the following - each dependency is scoped to the depending component:

```js
// MyComponent.js
import Ractive from 'ractive'

export default Ractive.extend({ ... })

// MyOtherComponent.js
import Ractive from 'ractive'
import MyComponent from './path/to/MyComponent.js'

export default Ractive.extend({
  components: { MyComponent },
  ...
})

// YetAnotherComponent.js
import Ractive from 'ractive'
import MyComponent from './path/to/MyComponent.js'

export default Ractive.extend({
  components: { MyComponent },
  ...
})
```

It's not safe to assume everyone's using the global registry. What we can say for certain is `styleSet` is a static function that exists on the `Ractive` constructor as well as any constructor that extends from it.

**Pull request closed without merging**